worldeditadditions/lib/selection/cloud.lua

```diff
@@ -0,0 +1,48 @@
+--  ██████ ██       ██████  ██    ██ ██████
+-- ██      ██      ██    ██ ██    ██ ██   ██
+-- ██      ██      ██    ██ ██    ██ ██   ██
+-- ██      ██      ██    ██ ██    ██ ██   ██
+--  ██████ ███████  ██████   ██████  ██████
+worldeditadditions.add_pos = {}
+local wea = worldeditadditions
+function worldeditadditions.add_point(name, pos)
```

Comment:
I feel the name of this function is too generic to be attached to the main `worldeditadditions` object. Perhaps it could be more descriptive? What about `sadditive_add` perhaps?

Reply:
`wea.selectors.add_point`?

Reply:
`wea.selection.add_point`?